web/ViewModels/PropertyInput/PropertyInputModelDescription.cs

```diff
@@ -1,30 +1,30 @@
-﻿using System.ComponentModel.DataAnnotations;
-using BricksAndHearts.Database;
-using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
-
-namespace BricksAndHearts.ViewModels.PropertyInput;
-
-public class PropertyInputModelDescription : PropertyInputModelBase
-{
-    [Required]
-    public string? Description { get; set; }
-
-    [ValidateNever]
-    public override string PreviousAction { get; set; } = "PropertyInputStepThreeDetails";
-
-    public override void InitialiseViewModel(PropertyDbModel property)
-    {
-        base.InitialiseViewModel(property);
-        Step = 4;
-        Title = "Property Description";
-        Description = property.Description;
-    }
-
-    public override PropertyViewModel FormToViewModel()
-    {
-        return new PropertyViewModel
-        {
-            Description = Description
-        };
-    }
+﻿using System.ComponentModel.DataAnnotations;

```

Comment:
The diff is making it look like this whole file has changed and I'm not sure why - this happened on one of Claire's PRs earlier and I believe she resolved it with Oskar. May be worth checking with Claire how this was fixed